src/components/Header/hamburger.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import classNames from 'classnames/dedupe';
+
+import 'hamburgers/dist/hamburgers.min.css';
+
+const Hamburger = ({onClick, isActive}) => {
+    let hamburgerClass = classNames('hamburger', 'is-block', 'is-paddingless', 'hamburger--spring', {'is-active': isActive});
+    return(
+        <div className={hamburgerClass} type="button" onClick={onClick}>
```

Comment:
It is really cool that you don't use logic or other long expressions in such places.